cmd/nfsmount/nfsmount.go

```diff
@@ -0,0 +1,50 @@
+// NFS mount is only needed for macOS since it has no
+// support for FUSE-based file systems
+//go:build darwin && !cmount
+// +build darwin !cmount
+
+package nfsmount
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os/exec"
+	"runtime"
+
+	"github.com/rclone/rclone/cmd/mountlib"
+	"github.com/rclone/rclone/cmd/serve/nfs"
+	"github.com/rclone/rclone/vfs"
+)
+
+func init() {
+	mountlib.NewMountCommand("mount", false, mount)
+	mountlib.AddRc("mount", mount)
+}
+
+func mount(VFS *vfs.VFS, mountpoint string, opt *mountlib.Options) (asyncerrors <-chan error, unmount func() error, err error) {
+	s, err := nfs.NewServer(context.Background(), VFS, &nfs.Options{})
```

Comment:
Does this choose a random port? If not it probably should do.

Reply:
If the port is not explicitly specified in options, then a random port is chosen automatically by TCP/IP library. in the subsequent lines. the actual port is extracted from server object and is passed to the mount command.

Reply:
Great